source/patterns/@aws-solutions-constructs/aws-iot-s3/README.md

```diff
@@ -64,6 +64,7 @@ _Parameters_
 |loggingBucketProps?|[`s3.BucketProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.BucketProps.html)|Optional user provided props to override the default props for the S3 Logging Bucket.|
```

Comment:
If the construct accepts IBucket instead of bucket, then the name of the attribute needs to be existingBucketInterface (and only the s3BucketInterface property is populated)